examples/v1beta1/hp-tuning/pbt.yaml

```
@@ -0,0 +1,57 @@
+apiVersion: kubeflow.org/v1beta1
+kind: Experiment
+metadata:
+  name: random-experiment
+  namespace: ''
```

Let's keep kubeflow namespace, to be consistent with other examples.
```suggestion
  namespace: kubeflow
```